unliftio/src/UnliftIO/Temporary.hs

```diff
@@ -4,6 +4,42 @@
 -- Strongly inspired by\/stolen from the <https://github.com/feuerbach/temporary> package.
 --
 -- @since 0.1.0.0
+--
+-- === __Copyright notice:__
```

Comment:
To avoid making the docs very large, this section is collapsible. It will be hidden by default when the user opens the documentation for this module. I assume this is sufficient for the terms of the license, since it is "provided" with the documentation.